spot_rl_experiments/spot_rl/models/yolov8predictor.py

```diff
@@ -0,0 +1,398 @@
+# Code contains minimal ports from the YOLOV8 official github repo & the example such that we don't break habitat lab environment,
```

Comment:
could you please explain why we need this function? Also could you please clean up the print statement?